docs/source/TQNet_Machine_Learning/layers/Custom Parallel Hybrid Network (CPHN) Layer.md

```diff
@@ -0,0 +1,82 @@
+# Custom Parallel Hybrid Network (CPHN) Layer
```

Comment:
how is this file supposed to be auto-used? You need to also link it in the actual rst file.